config/initializers/spree.rb

```diff
@@ -18,6 +19,13 @@
   config.shipping_instructions = true
   config.address_requires_state = true
 
+  # Set defaults for testing
+  if Rails.env.test?
+    config.stripe_connect_enabled = false
+    config.enable_embedded_shopfronts = false
+    config.products_require_tax_category = false
+  end
```

Comment:
Shouldn't this go in `config/environments/test.rb`? No need for an if there and that's the place where I'd expect all this kind of settings to be found.

Reply:
Yeah, that’s fine. I wasn’t sure so I went with this, but happy to to test.rb instead.